src/Blocks/components/load-more/components/load-more-options.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { ToggleControl, TextControl, RangeControl } from '@wordpress/components';
+import { checkAttr } from '@eightshift/frontend-libs/scripts/helpers';
+import manifest from '../manifest.json';
+
+const { options, title } = manifest;
+
+export const LoadMoreOptions = (attributes) => {
+	const {
+		setAttributes,
+		componentName = manifest.componentName,
+		loadMoreShowControls = true,
+		label = title,
+		loadMoreUse = checkAttr('loadMoreUse', attributes, manifest, componentName),
+		loadMoreUrl = checkAttr('loadMoreUrl', attributes, manifest, componentName),
+		loadMoreUsePagination = checkAttr('loadMoreUsePagination', attributes, manifest, componentName),
+		loadMoreItemsPerPage = checkAttr('loadMoreItemsPerPage', attributes, manifest, componentName),
+		loadMoreItemsPerPageParameterName = checkAttr('loadMoreItemsPerPageParameterName', attributes, manifest, componentName),
+		loadMoreStartItem = checkAttr('loadMoreStartItem', attributes, manifest, componentName),
+		loadMoreStartItemParameterName = checkAttr('loadMoreStartItemParameterName', attributes, manifest, componentName),
+	} = attributes;
+
+
+	if (!loadMoreShowControls) {
+		return null;
+	}
+
+	return (
+		<Fragment>
+
+			{label &&
+				<h3 className={'options-label'}>
+					{label}
+				</h3>
+			}
+
+			<ToggleControl
+				label={sprintf(__('Use %s', 'Unicorns'), label)}
+				checked={loadMoreUse}
+				onChange={(value) => setAttributes({ [`loadMoreUse`]: value })}
+			/>
+
+
+			{loadMoreUse &&
+				<Fragment>
+
+				<TextControl
+					label="URL"
+					value={loadMoreUrl ?? ''}
+					onChange={(value) => setAttributes({ [`loadMoreUrl`]: value })}
+					/>
+
+				{loadMoreUsePagination && <hr />}
```

Comment:
dont to this

Reply:
Can you clarify on which line exactly you meant here?
`onChange={(value) => setAttributes({ ['loadMoreUrl']: value })}`
or
`{loadMoreUsePagination && <hr />}` ?